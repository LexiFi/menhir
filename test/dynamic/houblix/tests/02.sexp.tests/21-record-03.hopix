let t = { x = 1, y = "FooBaR" }
let u = { z = t, some_more = t }
let test = print_string ("Recording succeeds.\n")