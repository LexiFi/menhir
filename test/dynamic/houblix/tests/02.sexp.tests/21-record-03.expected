(((value(DefineValue(SimpleValue(((value(Id t))(position((start_p(""1 0 4))(end_p(""1 0 5)))))()((value(Record((((value(LId x))(position((start_p(""1 0 10))(end_p(""1 0 11)))))((value(Literal((value(LInt 1))(position((start_p(""1 0 14))(end_p(""1 0 15)))))))(position((start_p(""1 0 14))(end_p(""1 0 15))))))(((value(LId y))(position((start_p(""1 0 17))(end_p(""1 0 18)))))((value(Literal((value(LString FooBaR))(position((start_p(""1 0 28))(end_p(""1 0 29)))))))(position((start_p(""1 0 28))(end_p(""1 0 29)))))))()))(position((start_p(""1 0 8))(end_p(""1 0 31)))))))))(position((start_p(""1 0 0))(end_p(""1 0 31)))))((value(DefineValue(SimpleValue(((value(Id u))(position((start_p(""2 32 36))(end_p(""2 32 37)))))()((value(Record((((value(LId z))(position((start_p(""2 32 42))(end_p(""2 32 43)))))((value(Variable((value(Id t))(position((start_p(""2 32 46))(end_p(""2 32 47)))))()))(position((start_p(""2 32 46))(end_p(""2 32 47))))))(((value(LId some_more))(position((start_p(""2 32 49))(end_p(""2 32 58)))))((value(Variable((value(Id t))(position((start_p(""2 32 61))(end_p(""2 32 62)))))()))(position((start_p(""2 32 61))(end_p(""2 32 62)))))))()))(position((start_p(""2 32 40))(end_p(""2 32 64)))))))))(position((start_p(""2 32 32))(end_p(""2 32 64)))))((value(DefineValue(SimpleValue(((value(Id test))(position((start_p(""3 65 69))(end_p(""3 65 73)))))()((value(Apply((value(Variable((value(Id print_string))(position((start_p(""3 65 76))(end_p(""3 65 88)))))()))(position((start_p(""3 65 76))(end_p(""3 65 88)))))((value(Literal((value(LString"Recording succeeds.\n"))(position((start_p(""3 65 112))(end_p(""3 65 113)))))))(position((start_p(""3 65 89))(end_p(""3 65 114)))))))(position((start_p(""3 65 76))(end_p(""3 65 114)))))))))(position((start_p(""3 65 65))(end_p(""3 65 114))))))
